### ISSUE TITLE
Debian postinst script still not robust

### ISSUE THREAD

**Issue by BHSPitMonkey** [artefactual/archivematica-storage-service#1]
After having tried to install 0.2.2ppa8.deb yesterday (before the fixes were pushed out), I attempted to apt-get purge the package and try it again today (ppa11). With some parts of the filesystem not left exactly as the script expects, it fails unnecessarily:

```
Setting up archivematica-storage-service (0.2.2ppa11) ...
creating archivematica user
User archivematica exists
creating django secret key
[redacted]
creating symlink in /usr/lib/archivematica
ln: failed to create symbolic link `/usr/lib/archivematica/storage-service/storage_service': File exists
mv: cannot move `/var/archivematica/storage-service/static' to `/usr/lib/archivematica/storage-service/static/static': Directory not empty
mv: cannot move `/var/archivematica/storage-service/templates' to `/usr/lib/archivematica/storage-service/templates/templates': Directory not empty
configuring django database and static files
/var/lib/dpkg/info/archivematica-storage-service.postinst: 33: /var/lib/dpkg/info/archivematica-storage-service.postinst: /usr/share/python/archivematica-storage-service/bin/python: not found
/var/lib/dpkg/info/archivematica-storage-service.postinst: 34: /var/lib/dpkg/info/archivematica-storage-service.postinst: /usr/share/python/archivematica-storage-service/bin/python: not found
updating directory permissions
rm: cannot remove `/tmp/storage-service.log': No such file or directory
dpkg: error processing archivematica-storage-service (--configure):
 subprocess installed post-installation script returned error exit status 1
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by mistydemeo** [comment]
Sorry for the radio silence here, but: this is fixed in the `dev/issue-xxxx-packaging` branch, which will be merged in preparation for the upcoming 0.4.0 release. This bug shouldn't affect future Storage Service versions.

Sorry for all the inconvenience this caused - I'm aware the `postinst` issue was a major pain.

**Comment by BHSPitMonkey** [comment]
Thanks for the update.
